testsuite/MDAnalysisTests/utils/test_qcprot.py

```diff
@@ -134,14 +135,15 @@ def test_CalcRMSDRotationalMatrix():
     aligned_rmsd = rmsd(frag_br.T, frag_a)
     #print 'rmsd after applying rotation: ',rmsd
 
-    assert_almost_equal(aligned_rmsd, 0.719106, 6, "RMSD between fragments A and B does not match excpected value.")
+    assert aligned_rmsd == approx(0.719106, abs=1e-6), ("RMSD between "
+           "fragments A and B does not match excpected value.")
 
     expected_rot = np.array([
         [0.72216358, -0.52038257, -0.45572112],
         [0.69118937, 0.51700833, 0.50493528],
         [-0.0271479, -0.67963547, 0.73304748]])
-    assert_almost_equal(rot.reshape((3, 3)), expected_rot, 6,
-                        "Rotation matrix for aliging B to A does not have expected values.")
+    assert_allclose(rot.reshape((3, 3)), expected_rot, rtol=0, atol=1e-6,
+                    err_msg="Rotation matrix for aliging B to A does not have expected values.")
```

Comment:
even though historical, let's fix the PEP8 issue here
```suggestion
    errmsg="Rotation matrix for aliging B to A does not have expected values."
    assert_allclose(rot.reshape((3, 3)), expected_rot, rtol=0, atol=1e-6,
                    err_msg=errmsg)
```